nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -627,6 +627,10 @@
   `goModules`, `modRoot`, `vendorHash`, `deleteVendor`, and `proxyVendor` are now passed as derivation attributes.
   `goModules` and `vendorHash` are no longer placed under `passthru`.
 
+- `buildFlags`/`buildFlagsArray` on `buildGoModule` have been deprecated. 24.11 is the last release where `buildGoModule` accepts these flags (while throwing a warning).
```

Comment:
I think it might make sense to show what's the alternative here (or to link the relevant section of the manual).